core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -432,26 +432,23 @@ private void checkTempBlockOwnedBySession(long sessionId, long blockId)
    */
   private void abortBlockInternal(long sessionId, long blockId) throws BlockDoesNotExistException,
```

Comment:
the javadoc already says 
`Since a temp block is "private" to the writer, this requires no previously acquired lock.`